pkg/common/plugin/httpchallenge/httpchallenge_test.go

```diff
@@ -79,13 +79,14 @@ func TestValidateChallenge(t *testing.T) {
 			}))
 			defer func() { testServer.Close() }()
 
-			transport := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
The DefaultTransport honors HTTP_PROXY env vars. We should actually start from a new Transport.